test/jasmine/tests/histogram2d_test.js

```diff
@@ -91,8 +91,7 @@ describe('Test histogram2d', function() {
 
             // TODO: even though the binning is done on non-uniform bins,
             // the display makes them linear (using only y0 and dy)
-            // when we sort out https://github.com/plotly/plotly.js/issues/1151
-            // lets also make it display the bins with nonuniform size
+            // Can we also make it display the bins with nonuniform size?
```

Comment:
Removed from the scope of this PR because it turned out to be more of a pain than I thought, and it's a really minor effect visually.